tests/kernels/test_hyperobolic_matern.py

```diff
@@ -0,0 +1,36 @@
+import geomstats.visualization as visualization
```

Comment:
File name to change to "test_hyperbolic_matern.py" (hyperbolic instead of hyperobolic)